examples/fluentd-vrli-plugin-photon-tdnf.dockerfile

```diff
@@ -0,0 +1,38 @@
+# Fluentd plugin for VMware Log Insight
+# 
+# Copyright 2019 VMware, Inc. All Rights Reserved. 
+# 
+# This product is licensed to you under the MIT license (the "License").  You may not use this product except in compliance with the MIT License.  
+# 
+# This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file. 
+# 
+# SPDX-License-Identifier: MIT
+
```

Comment:
Thanks for the contribution @shadjiiski. Can please you add a comment on the top for all 3 files explaining what they're for?